apps/wing-console/console/ui/src/features/console-logs.tsx

```diff
@@ -6,11 +6,11 @@ import {
 import { useTheme, ResourceIcon } from "@wingconsole/design-system";
 import { LogEntry } from "@wingconsole/server";
 import classNames from "classnames";
+import Linkify from "linkify-react";
```

Comment:
Need to add it to `console/ui/package.json`

Reply:
missed the file, sorry :)